tests/fixtures/nwchem/test-001/run.sh

```diff
@@ -3,4 +3,4 @@
 
 module add nwchem/702-g-485-ompi-110
 
-mpirun --allow-run-as-root -np 1 nwchem.inp &> nwchem.log
+mpirun --allow-run-as-root -np 1 nwchem.inp &> nwchem-sp-energy.log
```

Comment:
What does "sp" stand for?

Reply:
single point energy, so it is a snapshot of the energy at that specific geometry, there is no optimization of the structure involved.

Reply:
Maybe calling it just "nwchem-energy.log" or "nwchem-total-energy.log" would be better?

Reply:
Either that or a short comment here as a reminder would be fine.

Reply:
Changed the naming